aries-site/src/pages/components/header.js

```diff
@@ -18,19 +26,214 @@ const Header = () => (
+        </Text>
+        <BulletedList
+          items={[
+            `Provide the user with relevant, helpful actions regarding the 
+            application or page`,
+            'Layout smoothly and responsively across all browser widths',
+          ]}
+        />
+      </Subsection>
+      <Subsection name="App Header" level={3}>
+        <SubsectionText>
+          An app-level Header sets the context for an entire application. It
+          should contain the HPE or Aruba logo alongside text that provides the
+          application name on the left side of the Header.
+        </SubsectionText>
+        <Text weight="bold">
+          Here are some elements that are appropriate for an App Header:
+        </Text>
+        <BulletedList
+          items={[
+            'Application navigation structure',
+            'Profile Avatar button that leads to account information',
+            'Search field that searches entirety of the application',
+          ]}
+        />
+        <Text weight="bold">
+          These elements are not appropriate for an App Header:
+        </Text>
+        <BulletedList
+          items={[
+            `Actions or buttons that pertain specifically to the context of a 
+            single page`,
+            'Search field that only searches within the current page',
+          ]}
+        />
+        <SubsectionText>
+          If you have actions that pertain specifically to a single page,
+          consider adding a Page Header to your layout in addition to your App
+          Header.
+        </SubsectionText>
+      </Subsection>
+      <Subsection name="Page Header" level={3}>
+        <SubsectionText>
+          A page-level Header sets the context for a single page within an
+          application. It may contain the main heading of the page as well as
+          some actions pertaining to the specific page content.
+        </SubsectionText>
+        <SubsectionText>
+          A Page Header should never replace the App Header. It should be used
+          in addition to the App Header and placed below the App Header in the
+          layout. For more guidance, check out an example of{' '}
+          <Anchor
+            label="how to use a page-level header"
+            href="#header-for-a-single-page"
+          />
+          .
+        </SubsectionText>
+        <Text weight="bold">Page Header actions may include:</Text>
+        <BulletedList
+          items={[
+            `Actions or buttons that pertain specifically to the context of the 
+            current page`,
+            'Search field that only searches within the current page',
+          ]}
+        />
+      </Subsection>
+      <Subsection name="Accessibility" level={3} gap="small">
+        <SubsectionText>
+          When using Header as a navigation structure, use{' '}
+          <Anchor label="Buttons" href={nameToPath('Button')} /> as your
+          navigational elements and wrap them in a Nav component. This allows
```

Comment:
curious is Nav component to be coming ? for design system

Reply:
Right now it's just on Grommet, we don't have documentation of it on the site -- I think it could be helpful to add in after this first deadline.

Reply:
sounds good then yes leave it off